webapi/src/main/scala/org/knora/webapi/slice/admin/api/PermissionsEndpoints.scala

```diff
@@ -107,7 +93,7 @@ final case class PermissionsEndpoints(base: BaseEndpoints) extends PermissionsAD
     putPerrmissionsHasPermissions,
     putPermisssionsResourceClass,
     putPermissionsProperty
-  ).map(_.endpoint)
+  ).map(_.endpoint.tag("Admin Permissions"))
```

Comment:
Shouldn't these ba actually two comma separated tags? This is overall question for all tags in our repo.

Reply:
As discussed, when using a single tag with our naming convention the request will only show up once in the Swagger-UI.